docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,79 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Functions
+---------
+
+The connector exposes the following table functions:
+
+system.table_changes
+^^^^^^^^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries.
+This feature exposes row-level changes between two versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data
+written into the table.
+In addition to columns present in the table this function
+returns 3 additional columns containing metadata about how and when the row
+was changed::
+``_change_type``, ``_commit_version``, ``_commit_timestamp``.
+
+.. code-block:: sql
+
+    SELECT * FROM TABLE(system.table_changes(schema_name => 'test_schema', table_name => 'tableName', since_version => 0))
```

Comment:
format into multiple lines for readability please .. see other table function docs

Reply:
We try to follow the Mozilla style pretty closely - https://docs.telemetry.mozilla.org/concepts/sql_style.html 

and yes.. we can not claim to use it consistently .. but thats what we aim for

How about this approximation

```
SELECT 
  * 
FROM 
  TABLE(
    system.table_changes(
      schema_name => 'test_schema', 
      table_name => 'tableName', 
      since_version => 0
    )
  )
```

Your short version is nice and compact.. but it just doesnt separate all the different aspects visually so a beginner can look at it and know whats going on.